src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -662,6 +662,36 @@ internal static void SetCurrentThreadUiCulture(CultureInfo uiCultureInfo)
         #region Misc
 
         /// <summary>
+        /// Facade for Convert.ToBase64String(bytes, Base64FormattingOptions.InsertLineBreaks)
+        /// Inserts line breaks after every 76 characters in the string representation.
+        /// </summary>
+        internal static string ToBase64StringWithLineBreaks(byte[] bytes)
+        {
+#if CORECLR
+            // Inserts line breaks after every 76 characters in the string representation.
+            string encodedRawString = Convert.ToBase64String(bytes);
+            if (encodedRawString.Length <= 76)
+                return encodedRawString;
+            
+            StringBuilder builder = new StringBuilder(encodedRawString.Length);
+            int index = 0, remainingLen = encodedRawString.Length;
+            while (remainingLen > 76)
```

Comment:
This is new implementation based on the [`base64LineBreakPosition`](https://referencesource.microsoft.com/#mscorlib/system/convert.cs,48d6bc9cac8a26a0) from the source code of `[System.Convert]::ToBase64String(byte[] inArray, Base64FormattingOptions options)`.
This method is used in `Protect-CmsMessage` through `CmsUtils.Encrypt`. I believe it's well exercised by the tests.